Source/PlayerEx.cpp

```diff
@@ -8,8 +8,24 @@ struct OP2Player
 	struct {
 		unsigned int rlv :4;
 		unsigned int solarSat :4;
+		unsigned int sUnk0 : 1;
 		unsigned int edwardSat :4;

```

Comment:
Ahh, thank you for the bug fix. I just verified this and indeed the original code was off by 1 for the bitfield offset.